sdk/communication/AzureCommunicationChat/AzureCommunicationChat.xcodeproj/xcshareddata/xcschemes/AzureCommunicationChat.xcscheme

```diff
@@ -34,23 +34,8 @@
             isEnabled = "YES">
          </EnvironmentVariable>
          <EnvironmentVariable
-            key = "AZURE_COMMUNICATION_ENDPOINT"
-            value = ""
-            isEnabled = "YES">
-         </EnvironmentVariable>
-         <EnvironmentVariable
-            key = "AZURE_COMMUNICATION_TOKEN"
-            value = ""
-            isEnabled = "YES">
-         </EnvironmentVariable>
-         <EnvironmentVariable
-            key = "AZURE_COMMUNICATION_USER_ID_1"
-            value = ""
-            isEnabled = "YES">
-         </EnvironmentVariable>
-         <EnvironmentVariable
-            key = "AZURE_COMMUNICATION_USER_ID_2"
-            value = ""
+            key = "SDK_REPO_ROOT"
+            value = "/Users/travisprescott/repos/azure-sdk-for-ios"
```

Comment:
This is my Mac user directory and yes, it is intentional. Anyone who wants to run the test would need to update this with their own settings, but since the build artifact isn't "aware" of the source directory, it (seemingly) has to be hard coded like this for the recording functionality to work correctly.